### PR TITLE
Add resque support

### PR DESCRIPTION
**Pull request by kbaum**
Out of box support for resque. Current implementation will report to redis for every job execution.

### CODE REVIEW

File: lib/coverband/integrations/resque.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Resque.after_fork do |job|
+  Coverband.start
+end
+
+module Coverband
```
Comment (danmayer) [inline lib/coverband/integrations/resque.rb]: nice this is very clean, well done

**Comment by danmayer** [comment]
merge when ever, I am not concerned on the -0.2% test change

**Comment by kbaum** [comment]
Thanks! Do you think we can ask the company that was having the memory issues with reque to try this out?

**Comment by kbaum** [comment]
I noticed the resque test is sporadically failing.

https://github.com/danmayer/coverband/issues/178

I'll try to figure out that issue and then do an alpha release?

Thanks for the invite to coverband_demo!

**Comment by danmayer** [comment]
cool makes sense to try to fix test if we can first... I am working on another feature, but may or may not make 4.0.2 release. We will see how long it takes to get some more feedback about resque and if we can reproduce the memory lead.

I am working on adding Gem coverage support via an optional flag. Reported as a new tab on the UI that lists all gems loaded and basic percentage usage per gem.

**Comment by kbaum** [comment]
Sounds good. That sounds like an amazing feature! So many gems that go
unused in big rails projects.

**Pull request merged**